### PR TITLE
fix: fix circuit version consistency check

### PR DESCRIPTION
**Pull request by vladimir-trifonov**
fix: fix circuit version consistency check

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -19,9 +19,9 @@ futures = "0.3.29"
 
 # zk-evm dependencies
 plonky2 = "0.2.0"
-evm_arithmetization = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.0" }
-trace_decoder = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.0" }
-proof_gen = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.0" }
+evm_arithmetization = "0.1.2"
```
Comment (muursh) [inline Cargo.toml]: @BGluth you were running the 701 right?

**Review by Nashtare — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (muursh) [inline Cargo.toml]: Can we update these to the latest versions

Comment (vladimir-trifonov) [reply Cargo.toml]: I did, but some test added recently fails... I will check it tmr

**Review by muursh — APPROVED** [review]

**Review by Nashtare — APPROVED** [review]
LGTM!

**Review by BGluth — APPROVED** [review]

**Pull request merged**